pkg/trace/api/api.go

```diff
@@ -350,6 +350,14 @@ const (
 	// headerComputedStats specifies whether the client has computed stats so that the agent
 	// doesn't have to.
 	headerComputedStats = "Datadog-Client-Computed-Stats"
+
+	// headderDroppedP0Traces contains the number of P0 trace chunks dropped by the client.
+	// This value is used to adjust priority rates computed by the agent.
+	headerDroppedP0Traces = "Datadog-Client-Dropped-P0-Traces"
+
+	// headderDroppedP0Spans contains the number of P0 spans dropped by the client.
+	// This value is used to adjust priority rates computed by the agent.
```

Comment:
Unless I'm misreading the code, this value seems to only be used for metrics (increasing an atomic value) and not taken into account for sampling. Should we update the text?